src/olympia/templates/photon-footer.html

```diff
@@ -0,0 +1,73 @@
+<footer class="Footer">
+    <div class="Footer-wrapper">
+        <div class="Footer-mozilla-link-wrapper">
+            <a class="Footer-mozilla-link" href="https://mozilla.org/" title="Go to Mozilla's homepage"><span class="Icon Icon-mozilla Footer-mozilla-logo"><span class="visually-hidden">Go to Mozilla's homepage</span></span></a>
+        </div>
+
+        <section class="Footer-amo-links">
+            <h4 class="Footer-links-header"><a href="{{ url('home') }}">Add-ons</a></h4>
+
+            <ul class="Footer-links">
+                <li><a href="{{ url('pages.about') }}">About</a></li>
+
```

Comment:
Maybe these blanks lines in between list-items could be removed?

Reply:
The html is directly copied from addons.mozilla.org/ so yes, but there too then?